install.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# TODO(everyone): Keep this script simple and easily auditable.
+
+set -e
+
+if [ "$(uname -m)" != "x86_64" ]; then
+  echo "Error: Unsupported architecture $(uname -m). Only x64 binaries are available." 1>&2
+  exit 1
+fi
+
+if [ "$OS" = "Windows_NT" ]; then
```

Comment:
Does windows define `OS`?

Reply:
Not quite sure, although I suspect the Deno authors have tested this properly. Either way, I'll put together a Github actions test for this that runs on windows.

Reply:
Just verified this.